superset/assets/src/dashboard/components/HeaderActionsDropdown.jsx

```diff
@@ -136,9 +141,11 @@ class HeaderActionsDropdown extends React.PureComponent {
           {t('Force refresh dashboard')}
         </MenuItem>
         <RefreshIntervalModal
-          onChange={refreshInterval =>
-            startPeriodicRender(refreshInterval * 1000)
-          }
+          refreshFrequency={refreshFrequency}
+          onChange={refreshInterval => {
```

Comment:
we prefer to avoid use arrow function or bind function in render's event handler. here is why:
https://medium.freecodecamp.org/react-pattern-extract-child-components-to-avoid-binding-e3ad8310725e

Usually we create reusable event handler in constructor. see `this.changeCss` as example.